components/infobox/wikis/heroes/infobox_league_custom.lua

```diff
@@ -60,6 +61,19 @@ function CustomInjector:parse(id, widgets)
 	return widgets
 end
 
+function CustomInjector:addCustomCells(widgets)
+	table.insert(widgets, Cell{
+		name = 'Teams',
+		content = {(_league.args.team_number or '') .. (_league.args.team_slots and ('/' .. _league.args.team_slots) or '')}
+	})
+	return widgets
+end
+
+function CustomLeague:addToLpdb(lpdbData, args)
+	lpdbData.participantsnumber = args.team_number
```

Comment:
isn't that already done via commons?
(am on phone so shit to check)

Reply:
It is not set via the Custom (obviously). From what i can tell, the following is set by Commons:
``header``
``series``
``organizers``
``sponsors``
``gamesettings``
``type``
``location``
``venue``
``format``
``dates``
``liquipediatier``
``links``

Reply:
> isn't that already done via commons? (am on phone so shit to check)

Having tested removal of the wiki custom adding this to LPDB, the data is not being accurately stored via LPDB 
![image](https://user-images.githubusercontent.com/58013431/222257089-582109fd-07f4-43fc-80e4-77dfdde35d54.png)
This remained constant after both a null edit and purge.

Reply:
ah right we only use it to determine the category